rollup.config.js

```diff
@@ -0,0 +1,31 @@
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+
+import typescript from '@rollup/plugin-typescript';
+import includePaths from 'rollup-plugin-includepaths';
```

Comment:
this plugin is really old and not part of the `@rollup` namespace, but I could not find another way.